test/table.js

```diff
@@ -46,7 +48,9 @@ function createTestTable(testTableData, classTags="") {
   const tableRows = tableBody.querySelectorAll("tr");
   const testIfSortedList = [];
   for (let [i, tr] of tableRows.entries()) {
-    testIfSortedList.push(tr.querySelectorAll("td").item(0).innerHTML);
+    if(tr.style.display !== 'none'){
```

Comment:
Looks like it could do with being run through a linter like Prettier. However this is not your fault; I should setup linting for this project...